support-frontend/assets/components/headers/header/header.jsx

```diff
@@ -6,6 +6,7 @@ import React, { Component, type Node } from 'react';
 
```

Comment:
the changes in this file are to add the banner, as it was hard to debug, but it would be nice to have this feature back as we had it in subs and membership frontend.  Easy to remove if we don't like it though.

Reply:
also, if we remove this splash banner, we would be able to go back to boolean for isTestUser/uatMode.

The reason this is in one PR is that whether we do this or not affects whether we use boolean or option<string> for test user mode for this new code.
If we did this in a later PR, we would have to rewrite all the test user code I just added to use new types, which would add a lot more work.